internal/build/build.go

```diff
@@ -11,11 +11,10 @@ import (
 // Much like android and ios are already used.
 const BuildPath = "go"
 
-// OutputDirectoryPath returns the path where the go-flutter binary and flutter
-// binaries blobs will be stored for a particular platform.
+// buildDirectoryPath returns the path in `BuildPath`/build.
 // If needed, the directory is create at the returned path.
-func OutputDirectoryPath(targetOS string) string {
-	outputDirectoryPath, err := filepath.Abs(filepath.Join(BuildPath, "build", "outputs", targetOS))
+func buildDirectoryPath(targetOS, path string) string {
```

Comment:
Missing type for `targetOS`

Reply:
In golang `targetOS, path string` == `targetOS string, path string`